.eleventy.js

```diff
@@ -58,10 +60,30 @@ module.exports = function (eleventyConfig) {
   }
   eleventyConfig.setLibrary(
     'md',
-    markdownIt({ html: true }).use(markdownItAnchor, {
-      permalink: true,
-      renderPermalink,
+    markdownIt({
+      html: true,
+      linkify: true,
+      replaceLink: (link, env) => {
+        // Convert relative links to absolute links
+        const base = process.env.MODE === 'gh_pages' ? '/liquid' : ''
+        if (link.startsWith('./')) {
+          const splitted = env.page.url.split('/')
+          splitted.splice(splitted.length - 1, 0, link.substr(2))
+          return base + splitted.join('/')
+        }
+        if (link.startsWith('../')) {
+          const splitted = env.page.url.split('/')
+          splitted.splice(splitted.length - 2, 1, link.substr(3))
+          return base + splitted.join('/')
+        }
```

Comment:
```suggestion
        const splitted = env.page.url.split('/')
        if (link.startsWith('./')) {
          splitted.push(link.substr(2))
          return base + splitted.join('/')
        }
        if (link.startsWith('../')) {
          splitted.splice(splitted.length - 2, 1, link.substr(3))
          return base + splitted.join('/')
        }
```

Reply:
Keeping as is. the variable is only needed if the condition resolves to true.

Reply:
```suggestion
        if (link.startsWith('./')) {
          const splitted = env.page.url.split('/')
          splitted.push(link.substr(2))
          return base + splitted.join('/')
        }
        if (link.startsWith('../')) {
          const splitted = env.page.url.split('/')
          splitted.splice(splitted.length - 2, 1, link.substr(3))
          return base + splitted.join('/')
        }
```
Then at least do this...

Reply:
This may result in links such as this one: `/components/?yolo#something/ld-button`